README.md

```diff
@@ -37,23 +37,4 @@ script that is helpful for running the examples:
 ./run <example>  # e.g. ./run nodejs
 ```
 
-## Steps to your first API call
-
-### 1. Get the access token
-
-Every request towards our APIs that requires end user authentication requires an access token.
-To obtain this token send a post request to `/tokens` with the SSN in the payload. Example SSNs
-can be fetched from the test-customers endpoint which only requires a API-key from the portal.
-
-### 2. Get customer details
-
-This request will retrieve information about a dummy customer from our "Customers" endpoint.
-This is an example of a how to do a GET request towards our APIs, you will easily adapt the
- code for all other endpoints using the GET method.
-
-[developer.dnb.no]: https://developer.dnb.no
```

Comment:
Was this removed by accident? These links are still in use.

Reply:
Good catch. I was probably a bit too greedy here.

Reply:
Re-added in f https://github.com/DNBbank/getting-started/pull/62/commits/f2304dc3df12a23aad88c84b05637d9fb0f97323.